src/ogd/games/BLOOM/features/CountyLatestMoney.py

```diff
@@ -0,0 +1,51 @@
+# import libraries
+from typing import Any, Dict, List, Optional
+from ogd.core.generators.Generator import GeneratorParameters
+from ogd.core.models.Event import Event
+from ogd.core.models.enums.ExtractionMode import ExtractionMode
+from ogd.core.models.FeatureData import FeatureData
+
+# import PerCountyFeature
+from ogd.core.generators.extractors.PerCountFeature import PerCountFeature
+from ogd.games.BLOOM.features.PerCountyFeature import PerCountyFeature
+
+class CountyLatestMoney(PerCountyFeature):
+    def __init__(self, params: GeneratorParameters):
+        super().__init__(params=params)
+        self.latest_money: Dict[str, Optional[int]] = {county: None for county in self.COUNTY_LIST}
+
+    # *** IMPLEMENT ABSTRACT FUNCTIONS ***
+    @classmethod
+    def _eventFilter(cls, mode: ExtractionMode) -> List[str]:
+        return ["switch_county"]
+
+    @classmethod
+    def _featureFilter(cls, mode: ExtractionMode) -> List[str]:
+        return []
+
+    def _validateEventCountIndex(self, event: Event):
```

Comment:
Because this feature is inheriting from `PerCountyFeature`, it should not implement `_validateEventCountIndex`. This can be removed.